cmd/pilot-agent/main.go

```diff
@@ -51,6 +52,8 @@ var (
 	connectTimeout         time.Duration
 	statsdUDPAddress       string
 	proxyAdminPort         int
+	controlPlaneAuthPolicy string
```

Comment:
I still believe this should be called 'allowInsecure', but we can review the API/config naming in a different forum...